### PR TITLE
SwiftLint warnings fixed!

### PR DESCRIPTION
**Pull request by wojtek717**

### CODE REVIEW

File: kpz-mpk/kpz-mpk/AppDelegate.swift [hunk 1]
```diff
@@ -10,28 +10,26 @@ import UIKit
...
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
         return true
     }
 
     // MARK: UISceneSession Lifecycle
 
-    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession,
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/AppDelegate.swift]: https://github.com/raywenderlich/swift-style-guide#function-declarations
```For functions with long signatures, put each parameter on a new line and add an extra indent on subsequent lines```

File: kpz-mpk/kpz-mpk/AppDelegate.swift [hunk 2]
```diff
@@ -10,28 +10,26 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
-
-
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/AppDelegate.swift]: You can get rid of all comments

File: kpz-mpk/kpz-mpk/AppDelegate.swift (hunk 1 above)
Comment (leszko11) [inline kpz-mpk/kpz-mpk/AppDelegate.swift]: https://github.com/raywenderlich/swift-style-guide#function-declarations
```For functions with long signatures, put each parameter on a new line and add an extra indent on subsequent lines```

**Comment by wojtek717** [comment]
@leszko11 Poprawiłem do style guid'a ale znowu mam problem z bitrise. Prosiłbym abyś zerknął w wolnej chwili.

**Comment by wojtek717** [comment]
[Project setup](https://trello.com/c/8T439rgM/16-project-setup)

**Pull request merged**